Library/Homebrew/extend/os/mac/search.rb

```diff
@@ -36,8 +36,10 @@ def search_casks(string_or_regex)
         results = cask_tokens.extend(Searchable)
                              .search(string_or_regex)
 
-        results |= DidYouMean::SpellChecker.new(dictionary: cask_tokens)
-                                           .correct(string_or_regex)
+        if results.empty?
```

Comment:
I think doing `results |=` would be preferable to add more results even if there's only one. Thoughts?

Reply:
> I think doing `results |=` would be preferable to add more results even if there's only one. Thoughts?

I agree. I'll repair it.